build.gradle

```diff
@@ -19,10 +19,16 @@ defaultTasks 'build','checkLicenses'
 
 def buildAliases = [
   'dev': [
+    'compileJava',
+    'compileTestJava',
```

Comment:
What do these lines do? from 22 to 26? were we not compiling before?

Reply:
At least some of these tasks weren't running before :D I basically want to have a dev task that will catch most simple errors before pushing up a branch and running CI.  So, I just copied some additional tasks from the circleci config to try to ensure that the basic checks were running.

Reply:
Purely for knowledge sharing, my habit is to use command-F9 to tell IntelliJ to build everything.  Picks up compilation errors nicely and in the IDE so it's easier for me to fix.

Still definitely worth having a gradle target that does sensible checks though.